developer-docs/bolt-api-servers.md

```diff
@@ -165,9 +166,18 @@ The metadata object is optional, and contains metadata about the task being run.
 - `parameters`: Object, *optional* - A JSON object whose keys are parameter names, and whose values are JSON objects with 2 keys:
     - `description`: String, *optional* - The parameter description.
     - `type`: String, *optional* - The type the parameter should accept.
+    - `sensitive`: Boolean, *optional* - Whether the task runner should treat the parameter value as sensitive
+    - `input_method`: String, *optional* - What input method should be used to pass params to task (stdin, environment, powershell)
 
 #### Files
-# TODO
+The files array contains details about the files the task needs as well as how to get them. Array items should be objects with the following keys:
+- `uri`: Object, *optional* - Information on how to request task files
+    - `path`: String, *optional* - Relative URI for requesting task content
+    - `params`: Object, *optional* - Query parameters for locating task data
+        - `environment`: String, *required* - Environment task files are in
```

Comment:
Is this required for the `uri` object? If so I think `params` is also required, since it contains environment